src/content/learn/index.md

```diff
@@ -379,59 +380,60 @@ button {
-Initially, each `MyButton`'s `count` state is `0`
+প্রাথমিকভাবে, প্রতিটি `MyButton` এর `count` state `0`
 
 </Diagram>
 
-<Diagram name="sharing_data_child_clicked" height={367} width={407} alt="The same diagram as the previous, with the count of the first child MyButton component highlighted indicating a click with the count value incremented to one. The second MyButton component still contains value zero." >
+<Diagram name="sharing_data_child_clicked" height={367} width={407} alt="আগেরটির মতো একই ডায়াগ্রাম, প্রথম চাইল্ড MyButton component এর গণনার সাথে হাইলাইট করা একটি ক্লিক নির্দেশ করে count এর মান একটিতে বৃদ্ধি করা হয়েছে। দ্বিতীয় MyButton component টিতে এখনও মান শূন্য রয়েছে।" >
 
-The first `MyButton` updates its `count` to `1`
+প্রথম `MyButton` তার `count` আপডেট করে `1` এ
 
 </Diagram>
 
 </DiagramGroup>
 
-However, often you'll need components to *share data and always update together*.
+যাইহোক, প্রায়ই *ডেটা শেয়ার করতে এবং সবসময় একসাথে আপডেট করার জন্য*  আপনার component গুলির প্রয়োজন হবে।
+
+উভয় `MyButton` component এ একই `count` প্রদর্শন করতে এবং একসাথে আপডেট করতে, আপনাকে পৃথক button গুলি থেকে state টিকে "উপরের দিকে" সবকটি সমন্বিত নিকটতম component এ সরাতে হবে।
 
-To make both `MyButton` components display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them.
+এই উদাহরণে, এটি `MyApp`:
 
-In this example, it is `MyApp`:
 
 <DiagramGroup>
 
-<Diagram name="sharing_data_parent" height={385} width={410} alt="Diagram showing a tree of three components, one parent labeled MyApp and two children labeled MyButton. MyApp contains a count value of zero which is passed down to both of the MyButton components, which also show value zero." >
+<Diagram name="sharing_data_parent" height={385} width={410} alt="ডায়াগ্রামে তিনটি component এর একটি tree দেখানো হয়েছে, একটি কে parent হিসেবে MyApp লেবেল করা হয়েছে এবং দুটি children কে MyButton লেবেল করা হয়েছে। MyApp-এ শূন্যের একটি গণনা মান রয়েছে যা উভয় MyButton component গুলিতে পাস করা হয়, যার মান শূন্যও দেখায়।" >
 
-Initially, `MyApp`'s `count` state is `0` and is passed down to both children
+প্রাথমিকভাবে, `MyApp`-এর `count` state `0` এবং উভয় children এর কাছে পাঠানো হয়
 
 </Diagram>
 
-<Diagram name="sharing_data_parent_clicked" height={385} width={410} alt="The same diagram as the previous, with the count of the parent MyApp component highlighted indicating a click with the value incremented to one. The flow to both of the children MyButton components is also highlighted, and the count value in each child is set to one indicating the value was passed down." >
+<Diagram name="sharing_data_parent_clicked" height={385} width={410} alt="আগেরটির মতো একই ডায়াগ্রাম, প্যারেন্ট MyApp কম্পোনেন্টের গণনা সহ হাইলাইট করা একটি ক্লিকের সাথে মান বৃদ্ধি করে। উভয় সন্তানের MyButton উপাদানগুলির প্রবাহও হাইলাইট করা হয়েছে, এবং প্রতিটি children এর গণনার মান একটিতে সেট করা হয়েছে যা নির্দেশ করে যে মানটি পাস হয়েছে।" >
```

Comment:
```suggestion
<Diagram name="sharing_data_parent_clicked" height={385} width={410} alt="আগেরটির মতো একই ডায়াগ্রাম, প্যারেন্ট MyApp কম্পোনেন্টের গণনা সহ হাইলাইট করা একটি ক্লিকের সাথে মান বৃদ্ধি করে। উভয় চাইল্ডের MyButton উপাদানগুলির প্রবাহও হাইলাইট করা হয়েছে, এবং প্রতিটি চাইল্ড এর গণনার মান একটিতে সেট করা হয়েছে যা নির্দেশ করে যে মানটি পাস হয়েছে।" >
```

Reply:
dimot nei, vaiya. I will add the changes as soon as possible. Thanks for your review and suggestions. You are doing great to make it awesome.

Reply:
Thanks a lot for your kind words vai. We are all doing our parts.

Reply:
Changes gulo add kore disi vai.